packages/plugins/quorum/src/eth_sendRawTransaction.ts

```diff
@@ -128,7 +124,7 @@ export default class EthSendRawTransaction {
     return params;
   }
 
-  private async ethSendRawTransactionResponse(params: ProxyResponseParams<string, any>) {
+  async interceptResponse(params: ProxyResponseParams<string, any>) {
```

Comment:
One thing I'd propose @emizzle , we should probably also introduce an interface next to the abstract `RpcInterceptor` class. The reason for that is that `RpcInterceptor` at the moment ensures certain APIs are implemented on all interceptor implementations. However, "external" plugins like this quorum one doesn't have access to `RpcInterceptor` and also probably shouldn't even inherit from it because it has things like `accounts, `web3, `nodeAccounts` etc that may not be needed in a custom rpc interceptor plugin.

We should probably ask all interceptors to implement an `IRpcInterceptor` instead and all interceptors inside `rpc-interceptors` just happen to inherit from `RpcInterceptor` for shared functionality.

Thoughts?